mne/epochs.py

```diff
@@ -1700,7 +1708,7 @@ class EpochsArray(Epochs):
     @verbose
     def __init__(self, data, info, events, tmin=0, event_id=None,
                  reject=None, flat=None, reject_tmin=None,
-                 reject_tmax=None, verbose=None):
+                 reject_tmax=None, baseline=(None, 0), verbose=None):
```

Comment:
Ahh, right. The default for this should be `None` then, otherwise it's an API change. It makes it different from `Epochs`, right? We might have to live with that difference so we don't break people's code.